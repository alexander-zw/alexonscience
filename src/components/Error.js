/**
 * The error component is basically a 404 page displayed when no other
 * content applies.
 */
import React from 'react';
import '../styles/index.css'

function Error() {
    return (
        <div className="text-div">
            <p>Error: Page does not exist!</p>
        </div>
    );
}

export default Error;
