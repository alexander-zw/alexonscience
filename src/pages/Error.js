import React from 'react';

const Error = () => {
    return (
       <div className="text-div">
            <p>Error: Page does not exist!</p>
       </div>
    );
}

export default Error;
