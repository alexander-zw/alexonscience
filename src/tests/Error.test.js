import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Error from '../pages/Error';

it('renders correctly', () => {
    const tree = renderer.create(<Error />).toJSON();
    expect(tree).toMatchSnapshot();
});
