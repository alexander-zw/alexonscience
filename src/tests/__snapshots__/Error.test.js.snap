// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p>
    Error: Page does not exist!
  </p>
</div>
`;
